.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "04:00"
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: org.jetbrains.kotlin:kotlin-reflect
+    versions:
+    - "> 1.4.32, < 2"
+  - dependency-name: com.nulab-inc:zxcvbn
+    versions:
+    - 1.3.4
+    - 1.3.5
+    - 1.3.6
+  - dependency-name: com.google.android.material:material
+    versions:
+    - 1.3.0
+    - 1.3.0-rc01
+  - dependency-name: androidx.fragment:fragment-ktx
+    versions:
+    - 1.3.0-rc02
+  - dependency-name: org.robolectric:robolectric
+    versions:
+    - "4.5"
+  - dependency-name: com.android.tools.build:gradle
+    versions:
+    - 4.1.2
```

Comment:
I think @DenBond7 should review the list of ignored packages in case some are accidental